module/skill/Look/src/Look.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef MODULE_SKILL_LOOK_HPP
+#define MODULE_SKILL_LOOK_HPP
+
+#include <nuclear>
+
+#include "extension/Behaviour.hpp"
+
+namespace module::skill {
+
+    class Look : public ::extension::behaviour::BehaviourReactor {
+    private:
+        /// @brief Stores configuration values
+        struct Config {
+            float smoothing_factor = 0.0;
+            float head_gain        = 0.0;
+            float head_torque      = 0.0;
+        } cfg;
+
+        /// @brief Whether to smooth the head movements
+        bool smooth = false;
```

Comment:
Isn't this part of config?

Reply:
No, but it seems my comment isn't clear enough so I'll fix that up. Whether smoothing is defined by the Look message. We store the last one so we know if we've switch between the two.